modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -174,9 +184,12 @@ one IP address pool. If you need to access the services from an external network
 configure load balancers and routers to manage the traffic.
 <10> You must set the platform to `none`. You cannot provide additional platform
 configuration variables for
-ifndef::ibm-z,ibm-z-kvm,ibm-power[your platform.]
-ifdef::ibm-z,ibm-z-kvm[IBM Z infrastructure.]
-ifdef::ibm-power[IBM Power Systems infrastructure.]
+ifndef::ibm-z,ibm-power,agnostic,rhv[bare metal]
+ifdef::ibm-z[IBM Z]
+ifdef::ibm-power[IBM Power]
+ifdef::agnostic[your]
+ifdef::rhv[RHV]
+infrastructure.
```

Comment:
This is still inadvertently affecting the rendering of some IBM Z docs (e.g., missed `ibm-z-kvm` tag). What are your thoughts on keeping this as close as possible to the original wording for all? Like this:
```
ifndef::ibm-z,ibm-z-kvm,ibm-power,rhv[your platform.]
ifdef::ibm-z,ibm-z-kvm[IBM Z infrastructure.]
ifdef::ibm-power[IBM Power Systems infrastructure.]
ifdef::rhv[RHV infrastructure.]
```

Reply:
OK, hope this covers all the cases.